bundles/org.openhab.binding.plex/src/main/java/org/openhab/binding/plex/internal/handler/PlexApiConnector.java

```diff
@@ -0,0 +1,407 @@
+            NotificationContainer notification = GSON.fromJson(msg, NotificationContainer.class);
+            if (notification != null) {
+                if (notification.getNotificationContainer().getType().equals("playing")) {
+                    if (listener != null) {
+                        listener.onItemStatusUpdate(
+                                notification.getNotificationContainer().getPlaySessionStateNotification().get(0)
+                                        .getSessionKey(),
+                                notification.getNotificationContainer().getPlaySessionStateNotification().get(0)
+                                        .getState());
+                    }
+                }
+            }
+        }
+
+        @OnWebSocketError
+        public void onError(Throwable cause) {
+            if (!isShutDown) {
+                logger.debug("WebSocket onError - reconnecting");
+                asyncWeb();
+            }
+        }
+
+        private void asyncWeb() {
+            ScheduledFuture<?> mySocketReconnect = socketReconnect;
+            if (mySocketReconnect == null || mySocketReconnect.isDone()) {
+                if (scheduler != null) {
+                    socketReconnect = scheduler.schedule(PlexApiConnector.this::connect, 5, TimeUnit.SECONDS); // WEBSOCKET_RECONNECT_INTERVAL_SEC,
+                }
+            }
+        }
+    }
+
+    /**
+     * Handles control commands to the plex player.
+     *
+     * Supports:
+     * - Play / Pause
+     * - Previous / Next
+     *
+     * @param command The control command
+     * @param playerID The ID of the Plex player
+     */
+    public void controlPlayer(Command command, String playerID) {
+        String commandPath = null;
+        if (command instanceof PlayPauseType) {
+            if (command.equals(PlayPauseType.PLAY)) {
+                commandPath = "/player/playback/play";
+            }
+            if (command.equals(PlayPauseType.PAUSE)) {
+                commandPath = "/player/playback/pause";
+            }
+        }
+
+        if (command instanceof NextPreviousType) {
+            if (command.equals(NextPreviousType.PREVIOUS)) {
+                commandPath = "/player/playback/skipPrevious";
+            }
+            if (command.equals(NextPreviousType.NEXT)) {
+                commandPath = "/player/playback/skipNext";
+            }
+        }
+
+        if (commandPath != null) {
+            try {
+                String url = "http://" + host + ":" + String.valueOf(port) + commandPath;
+                Properties headers = getClientHeaders();
+                headers.put("X-Plex-Target-Client-Identifier", playerID);
+                HttpUtil.executeUrl("GET", url, headers, null, null, REQUEST_TIMEOUT_MS);
+            } catch (Exception e) {
+                logger.error("An exception occurred trying to send command '{}' to the play player: {}", commandPath,
+                        e.getMessage());
```

Comment:
Bindings should only log to error if something severe happened, like the detection of a bug in your code. This could be warn. See this link for a description of the log levels: https://www.openhab.org/docs/developer/guidelines.html#f-logging